packages/bulk-edit/package.json

```diff
@@ -8,7 +8,8 @@
     "@theia/filesystem": "1.46.0",
     "@theia/monaco": "1.46.0",
     "@theia/monaco-editor-core": "1.72.3",
-    "@theia/workspace": "1.46.0"
+    "@theia/workspace": "1.46.0",
+    "tslib": "^2.6.2"
```

Comment:
We don't need this for every package. It should be enough as long as `@theia/application-package` depends on `tslib`. This only adds maintenance effort, as we need to keep track of this dependency for every package.